### DISCUSSION TITLE
Is it possible to use this library to render charts in html emails ?

### DISCUSSION THREAD

**Discussion by Sandeepv68** [ChartsCSS/charts.css#28] [Q&A] [answered]
As of now, most companies render charts on server and convert it to image and then attach it to mails. Will it work if we render chartscss directly to email, since they are just html mails.

**Comment by ramiy** [comment, marked as answer]
@SandeepVattapparambil Not tested yet. But please update if you will test.

From what I know, email clients use HTML4. **Chart.css** uses HTML tables to structure the data, and CSS to style the table. If the user can load the CSS file inside the email he will see the chart, otherwise the user will see a simple table. Note that some email clients block external requests to prevent spam and tracking.
